source/vecs/entity.d

```diff
@@ -854,7 +927,7 @@ unittest
 	assertEquals(Entity(0), em._entities.front);
 }
 
-@safe pure
+@system
```

Comment:
Why you changing those PRs to `@system`. This part of the API is `@safe` and you are using CTFE here, so it doesn't matter anyway

Reply:
Everything dependent on Signal will be @system and not pure. Setting a component emits onSet.
```d
auto e = em.gen(Foo(3, 5), Bar("str"));
```

The code above emits onSet. It's similar to:
```d
auto e = em.gen(); // @safe pure
em.set(e, Foo(3, 5), Bar("str")); // @system
```